internal/metrics/metrics.go

```diff
@@ -74,6 +78,30 @@ func init() {
 		},
 		[]string{"conditionKind"},
 	)
+
+	ConditionInKV = promauto.NewCounterVec(
```

Comment:
The plan here is to collect metrics on the conditions in the `active-conditions` KV.

You're right this is ideally a Gauge, although, as a Gauge - the collection code gets more involved - since the Gauge value has to be set to the total for each Condition  grouped by its `conditionKind` and `state` values.

With the Counter type instead we just increment the value as we're iterating through and we use [delta](https://prometheus.io/docs/prometheus/latest/querying/functions/#delta) in the visualisation to obtain the difference over time. Let me know if Im missing something here.

Reply:
the composites tracked by `active-conditions` don't have a `kind` per se. 

I think if you want to track active conditions, we're probably better served by having a small go-routine that enumerates the KV and sets the gauge on a cadence. My experience in using PromQL counters for a gauge is that the queries get weird and brittle quickly. We should keep this metric more intuitive.

Reply:
thats a fair point, I'll leave a note in the code and have it taken up separately